common/src/main/java/org/apache/rocketmq/common/BrokerConfig.java

```diff
@@ -376,12 +379,13 @@ public boolean isEnable() {
         }
     }
 
-    private MetricsExporterType metricsExporterType = MetricsExporterType.DISABLE;
+    private MetricsExporterType metricsExporterType = MetricsExporterType.LOGGER;
```

Comment:
Please do not change the default value.

Reply:
@aaron-ai Could you please approve running the ci? And we need to add a jul to sl4j bridge in [aliyunmq/rocketmq-shaded-slf4j-api-bridge](https://github.com/aliyunmq/rocketmq-shaded-slf4j-api-bridge/tree/master).

Reply:
Actually there is [an official bridge from SLF4j](https://www.slf4j.org/legacy.html#:~:text=endless%20loop.-,jul%2Dto%2Dslf4j%20bridge,-The%20jul%2Dto).

Reply:
> Actually there is [an official bridge from SLF4j](https://www.slf4j.org/legacy.html#:~:text=endless%20loop.-,jul%2Dto%2Dslf4j%20bridge,-The%20jul%2Dto), @lrybbs just try it out.

You are right, but we should make it compatible with shaded sl4j. Can you give some guide for @lrybbs?

Reply:
@aaron-ai Now, broker metrics and proxy metric could output to "rocketmqlogs" folder. Their names are "broker_metric.log" and "proxy_metric.log", respectively.